pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -823,6 +823,9 @@ type RemoteReadSpec struct {
 	TLSConfig *TLSConfig `json:"tlsConfig,omitempty"`
 	// Optional ProxyURL.
 	ProxyURL string `json:"proxyUrl,omitempty"`
+	// Configure whether HTTP requests follow HTTP 3xx redirects.
+	// Requires Prometheus v2.26.0 and above.
+	FollowRedirects bool `json:"followRedirects,omitempty"`
```

Comment:
Can you change the type to be `*bool` and add `//+optional` to it? This will allow us to leave the default value to Prometheus.

Reference: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required

Reply:
Thanks for the review @xiu,
sure, I've just committed the changes